appengine/Dockerfile

```diff
@@ -54,7 +55,7 @@ ENV PATH /rbenv/shims:/rbenv/bin:$PATH
 
 # Preinstall ruby runtimes.
 # The LAST version in the list is set as the default.
-ENV PREINSTALLED_RUBY_VERSIONS 2.0.0-p647 2.1.7 2.2.3
+ENV PREINSTALLED_RUBY_VERSIONS 2.0.0-p648 2.1.8 2.2.3 2.2.4
```

Comment:
@bmclean No, it hasn't been pushed to gcr.io yet. I was waiting for a second PR to go through to preinstall 2.3.0. Sorry for the confusion. You're right, we need to have a standard way to push and communicate releases.

I wasn't expecting deploys to fail outright when asking for a newer Ruby version. The intent was that requesting a ruby not in the preinstall list would work, but just be slower. I just realized though, that what's probably causing your failures is that the base image's rbenv and ruby-build installations are too old. That needs to be fixed in the gcloud sdk tool: "gcloud preview app deploy" command should git-pull the ruby-build directory to ensure it has the latest info at the time of deployment. I'll file an internal issue and make sure that gets fixed. Sorry for the hiccup and thanks for the report!